src/goalServices/goals.js

```diff
@@ -504,10 +505,6 @@ export async function createOrUpdateGoals(goals) {
       if (newGoal.status !== status) {
         newGoal.set({ status });
       }
-
-      if (!newGoal.createdVia || newGoal.createdVia !== createdVia) {
```

Comment:
Did we look into why we added this?

Reply:
We added it to account for goals that didn't have the createdVia yet, since we didn't write a big migration when we added the column. At this point, I feel that it is safe to discard.

Reply:
Specifically, the smartsheet imported goals didn't have createdVias